test/unit/liquid/drops/api_spec_drop_test.rb

```diff
@@ -34,6 +34,16 @@ def setup
     assert_equal @service.proxy.endpoint, api_spec.api_product_production_public_base_url
   end
 
+  test 'Returns fase for unpublished api spec' do
+    @spec.update(published: false)
+    assert_equal false, @spec.published?
```

Comment:
This tests the model itself, not the drop. So this would pass even before adding the method.